build/buildconstants/params_shared_vals.go

```diff
@@ -0,0 +1,112 @@
+//go:build !testground
+// +build !testground
+
+package buildconstants
+
+import (
+	"math/big"
+	"os"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	builtin2 "github.com/filecoin-project/specs-actors/v2/actors/builtin"
+
+	"github.com/filecoin-project/lotus/chain/actors/policy"
+)
+
+// /////
+// Storage
+
+const UnixfsChunkSize uint64 = 1 << 20
+const UnixfsLinksPerLevel = 1024
```

Comment:
FYI getting rid of these in https://github.com/filecoin-project/lotus/pull/12217